zone/zonedb.cpp

```diff
@@ -3910,6 +3913,34 @@ uint32 ZoneDatabase::GetCharacterCorpseItemAt(uint32 corpse_id, uint16 slot_id)
 	return item_id;
 }
 
+bool ZoneDatabase::LoadCharacterCorpseRezData(uint32 corpse_id, uint32 *exp, uint32 *gm_exp, bool *rezzable, bool *is_rezzed)
+{
+	std::string query = StringFormat(
+		"SELECT           \n"
+		"exp,             \n"
+		"gm_exp,		  \n"
+		"rezzable,		  \n"
+		"is_rezzed		  \n"
+		"FROM             \n"
+		"character_corpses\n"
+		"WHERE `id` = %u  LIMIT 1\n",
+		corpse_id
+	);
+
+	auto results = QueryDatabase(query);
+	uint16 i = 0;
+
+	for (auto& row = results.begin(); row != results.end(); ++row) {
+		*exp       = Strings::ToUnsignedInt(row[i++]);	// exp,
+		*gm_exp    = Strings::ToUnsignedInt(row[i++]);	// gm_exp,
+		*rezzable  = Strings::ToInt(row[i++]);			// rezzable
+		*is_rezzed = Strings::ToInt(row[i++]);			// is_rezzed
+
+		return true;
+	}
+	return false;
```

Comment:
Would prefer repositories after all the recent work to use them. (Obviously this came before those changes were made.)

Reply:
Updated to use repository method. I believe that is the correct way of utilizing it?